testing/renovation/platforms/declaration/data_grid_light.tsx

```diff
@@ -3,31 +3,57 @@ import {
   Component, ComponentBindings, JSXComponent, InternalState, Effect,
 } from '@devextreme-generator/declarations';
 import React from 'react';
+import { DeepPartial } from '../../../../js/core';
 import { DataGridLight, DataGridLightProps } from '../../../../js/renovation/ui/grids/data_grid_light/data_grid_light';
 import { Pager, PagerProps } from '../../../../js/renovation/ui/grids/data_grid_light/pager/pager';
 import { Paging, PagingProps } from '../../../../js/renovation/ui/grids/data_grid_light/paging/paging';
+import { Selection, SelectionProps } from '../../../../js/renovation/ui/grids/data_grid_light/selection/selection';
+
+// eslint-disable-next-line @typescript-eslint/no-type-alias
+export type OptionsType = DeepPartial<
+DataGridLightProps
+& { pager: PagerProps }
+& { selection: SelectionProps }
+& { paging: PagingProps }
+>;
 
 export const viewFunction = ({
-  options, pager, paging, setPageIndex, setPageSize,
+  options, pager, paging, selection, setPageIndex, setPageSize, setSelectedRowKeys,
 }: App): JSX.Element => (
   <DataGridLight
     id="container"
     dataSource={options.dataSource}
     columns={options.columns}
+    keyExpr={options.keyExpr}
   >
+    {paging.enabled && (
     <Paging
       enabled={paging.enabled}
       pageIndex={paging.pageIndex}
       pageIndexChange={setPageIndex}
       pageSize={paging.pageSize}
       pageSizeChange={setPageSize}
     />
+    )}
+
+    {/* {pager.visible && ( */}
```

Comment:
Is this comment necessary?

Reply:
Yes, if I uncomment it, angular tests will fail